### PR TITLE
Enterprise drive and output format improvements

### PR DESCRIPTION
**Pull request by kendmerry**
Add some changes, primarily to the Enterprise drive code, to do a few things:
o Make it easier to operate on drives that haven't had a password set yet.
o Make the initial setup code work without erring out on Seagate and HGST drives.
o Add an output mode without the time/date to be a little more readable.
o Make sedutil exit statuses consistent. In particular, it's important to know when we get SP_BUSY status back, so we can take action (reset the drive) to restore communication with the encryption processor.

I have tested this on FreeBSD, using the changes from @amotin with Seagate and HGST SED drives.

Sponsored by: Spectra Logic

### CODE REVIEW

File: Common/DtaDevEnterprise.cpp [hunk 1]
```diff
@@ -747,15 +854,15 @@ uint8_t DtaDevEnterprise::setLockingRange_SUM(uint8_t lockingrange, uint8_t lock
...
+	return DTAERROR_INVALID_PARAMETER;
 }
 uint8_t DtaDevEnterprise::enableUser(char * password, char * userid)
 {
 	LOG(D1) << "Entering DtaDevEnterprise::enableUser";
 	LOG(E) << "enableUser not implemented";
 	if (!password && !userid) { LOG(E) << "Formal Parameters"; }
 	LOG(D1) << "Exiting DtaDevEnterprise::enableUser()";
-	return 0xff;
+	return DTAERROR_INVALID_PARAMETER;
```
Comment (cristim) [inline Common/DtaDevEnterprise.cpp:865]: I'm confused, shouldn't this be DTAERROR_UNSUPORTED_LOCKING_RANGE?

I've seen some more such inconsistencies below, I'm afraid not to introduce some subtle bugs.

**Comment by ekoyle** [comment]
I think this would be a lot easier to review in a series of small pull requsts.

**Comment by cristim** [comment]
@kendmerry I'd really like to see this merged but I'm having a hard time reviewing it in the current state, and it's been pending for the review for too long.

I wonder, is there anyone who would be willing and able to get this reviewed as it is within a few more days?

Otherwise please consider splitting it in a series of PRs, one for each of the implemented features, which would be easier to review.

**Comment by kendmerry** [comment]
I’m on vacation this week, but I may have some time next week to work on splitting it up a bit.  It originally went in as separate changes, so I could just put in PRs for groups of those.

Ken
— 
Ken Merry
ken@kdm.org

**Comment by cristim** [comment]
It can wait a week longer, unless someone else is able to get it reviewed earlier in the current state

**Comment by cristim** [comment]
Closing PR, this fork is now obsolete since DTA resumed active development in the open.

**Pull request closed without merging**